src/main/java/de/zalando/aruha/nakadi/domain/Cursor.java

```diff
@@ -8,6 +8,7 @@
 public class Cursor {
 
     public static final String BEFORE_OLDEST_OFFSET = "BEGIN";
+    public static final String AFTER_NEWEST_OFFSET = "END";
```

Comment:
Since pointers are exclusive, e.g. if you have offsets [1, 2, 3, 4], and your cursors points to 2, then it should stream from 3 and 4.

That said, I would argue that the name of this constant should be `NEWEST_OFFSET`.